doc/source/development/rfc/rfc85_policy_code_additions.rst

```diff
@@ -0,0 +1,85 @@
+.. _rfc-85:
+
+===========================================================
+RFC 85: Policy regarding substantial code additions
+===========================================================
+
+======== ==================================================
+Authors: Howard Butler, Even Rouault
+======== ==================================================
+Started: 2022-01-17
+Status:  Development
+======== ==================================================
+
+Summary
+-------
+
+This document describes the policies that the GDAL project will apply to assess
+substantial code additions, typically new drivers, in particular coming for new
+contributors to the project.
+
+Motivation
+----------
+
+The GDAL project has historically been quite open to new code additions, including
+drivers that rely on proprietary licensed and/or closed-source SDKs (called
+"binary SDK" in the rest of this document). This approach is part
+of its strengths, but also comes at a price, with the software being larger
+and larger, with sometimes contributions being abandoned by their authors without
+new maintainers.
+
+Being a free & open-source project, the GDAL project will apply stronger scrutiny to
+code that require a binary SDK and encourage contributors to submit code additions
+that do not depend on such binary SDK or license such SDK under free & open-source
+license terms.
+
+Policy
+------
+
+- Drivers require a designated responsible contact, tracked in
+  https://github.com/OSGeo/gdal/wiki/Maintainers-per-sub-system
+
+- Contributions should follow other RFCs describing development rules, and come
+  with test scripts and sufficient documentation, covering usage and build instructions.
+
+- If the driver require a binary SDK not downloadable without cost, or that requires
+  a complicated registration process, the GDAL team is unlikely to support
+  driver inclusion.
+
```

Comment:
```suggestion
- GDAL supports multiple operating systems, and new drivers should support as many as
  is practical. At a minimum, drivers should work with the latest and OS-vendor supported
  releases of operating systems the driver is documented to work with.

```

My wording here should cover things like: "my driver/library only works on RHEL7/macOS 10.9/Windows 8" without necessarily requiring everything to work on macOS/Android/Windows/32bit Linux/etc